docs/parameters-and-payloads.md

```diff
@@ -34,11 +34,11 @@ Since we've supplied two objects here, the SDK automatically knows that you're s
 
 > ℹ️
 >
-> For background, the algorithm to determine if you're supplying a `body` or `metadata` for the first parameter runs a difference check against the payloads for either and if what you're supplying matches +25% of what is documented in the operations request body payload then we'll treat it as a `body` payload.
+> For background, the algorithm to determine if you're supplying a `body` or `metadata` for the first parameter runs a difference check against the payloads for either. If what you're supplying matches +25% of what is documented in the operation's request body payload, then we'll treat it as a `body` payload.
```

Comment:
Should "operation's" have an apostrophe in this context?

Reply:
I think so? The way I'm reading this, the request body payload is possessed by the operation